(GH-1894) Do not load projects from world-writable directories (#1944)
puppetlabs/bolt: documentation/bolt_project_directories.md

1. **User project directory:** If no directory is specified manually or found in
   a parent directory, the user project directory is used.

## World-writable project directories

> By default, Bolt will not load a project from a world-writable directory, as
> they present a potential security risk. If you attempt to load a project from
> a world-writable directory, Bolt will not load any content and will raise an

Reviewer: ```suggestion
By default, Bolt will not load a project from a world-writable directory, as
loading from a world-writable directory presents a potential security risk. 
If you attempt to load a project from a world-writable directory, Bolt will 
not load any content and will raise an
```

> By default, Bolt will not load a project from a world-writable directory, as
> they present a potential security risk. If you attempt to load a project from
> a world-writable directory, Bolt will not load any content and will raise an
> exception.

Reviewer: ```suggestion
By default, Bolt will not load a project from a world-writable directory, because
loading from a world-writable directory presents a potential security risk. 
If you attempt to load a project from a world-writable directory, Bolt will not load 
any content and will raise an exception.
```

If you wish to override this behavior and force Bolt to load a project from a
world-writable directory, you can set the `BOLT_PROJECT` environment variable

> to the project directory path.
> 

Reviewer: ```suggestion
to the project directory path.

For example, if you wanted to load a project named `my_project` from the world-writable directory
at `~/project/`, you would set the `BOLT_PROJECT` environment variable as:

```

```bash
export BOLT_PROJECT='~/project/world_writable'
```

> 

Reviewer: ```suggestion

> **Note:** Exported environment variables expire at the end of the current session. If you need 
> a more permanent solution, add the `export` line to your `~/.bashrc` or the relevant profile for
> the shell you're using.

If you want to use a world-writable directory for a single Bolt execution, set the environment
variable before the Bolt command:   
```
